server/pulp/server/managers/consumer/applicability.py

```diff
@@ -132,9 +132,12 @@ def regenerate_applicability_for_repos(repo_criteria):
         repo_ids = [r['id'] for r in repo_query_manager.find_by_criteria(repo_criteria)]
 
         for repo_id in repo_ids:
-            # Find all existing applicabilities for given repo_id
+            # Find all existing applicabilities for given repo_id. If batch size is not specified,
```

Comment:
Not a required change but consider moving these details to the issue and shortening this to something more direct for the developer. Maybe link directly to your comment with all this good detail. Something like:

`setting batch_size to 25 ensures the MongoDB cursor doesn't timeout. See https:/pulp.plan.... for more details.`